companies/1plus-agency.json

```diff
@@ -13,6 +13,12 @@
     "fax": "+49 6252 6990559",
     "email": "info@1plus-agency.com",
     "web": "https://www.1plus-agency.com/",
+    "required-elements": [
+        {
+            "desc": "abc",
+            "type": "email"
+        }
+    ],
```

Comment:
🚫 **[companylint]** <[name-in-required-elements](https://github.com/datenanfragen/data#required-elements)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Record has required elements but no `name` element.